addons/skin.estuary/xml/Variables.xml

```diff
@@ -965,4 +965,11 @@
 	<variable name="PVRInstanceName">
 		<value condition="Integer.IsGreater(PVR.ClientCount,1)">$INFO[ListItem.PVRClientName,[COLOR grey]$LOCALIZE[31137]:[/COLOR] ,]$INFO[ListItem.PVRInstanceName, (,)]</value>
 	</variable>
+	<variable name="VideoSubsStreamDialogItemLabelVar">
+		<value condition="String.IsEqual(ListItem.Property(stream.isforced),true)+String.IsEqual(ListItem.Property(stream.ishearingimpaired),true)">$LOCALIZE[39106], $LOCALIZE[39107]</value>
```

Comment:
@Hitcher about this var, example:
String.IsEqual(ListItem.Property(stream.isforced),true)
do you know if exists a bool comparison without use "string"?

Reply:
All bools are listed in https://github.com/xbmc/xbmc/blob/master/xbmc/guilib/guiinfo/GUIInfoLabels.h and I don't see anything relevant hence probably why a ListItem.Property is being used.